### ISSUE TITLE
Asset search UI fails -- Asset Catalog OMAS error

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#2426]
I have Egeria configured via the configure/start notebooks, and have also run the asset management notebook. These all run cleanly, with the end result of a configured egeria environment including a few csv assets

I launch the UI and try an asset search for CSV. However the UI reports an error (We are experiencing an unexpected error)

The root cause of the error appears to be:
```
2020-01-15 10:33:13.155 ERROR 1 --- [io-8443-exec-10] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is PropertyServerException{reportedHTTPCode=404, reportingClassName='org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler', reportingActionDescription='searchByType', errorMessage='OMAG-PLATFORM-404-002 The Asset Catalog OMAS service is not available on OMAG Server cocoMDS4 to handle a request from user faithbroker', reportedSystemAction='The system is unable to process the request.', reportedUserAction='Verify that the correct server is being called on the correct platform and that the requested service is configured to run there.  Once the correct environment is in place, retry the request.', reportedCaughtException=null, relatedProperties=null}] with root cause

org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException: OMAG-PLATFORM-404-002 The Asset Catalog OMAS service is not available on OMAG Server cocoMDS4 to handle a request from user faithbroker
```
Looking at configured services:
```
➜  ~ curl http://egeriadev.eu-gb.containers.appdomain.cloud:30081/open-metadata/platform-services/users/garygeeke/server-platform/servers/cocoMDS4/services
{"relatedHTTPCode":200,"serverName":"cocoMDS4","serverServicesList":["Open Metadata Repository Services (OMRS)","Connected Asset Services","Community Profile OMAS","Asset Consumer OMAS","Glossary View OMAS","OMAG Server Operational Services","Data Science OMAS","Asset Catalog OMAS "]}%    ➜  ~
```
And indeed this is supported by the docs, plus the output from server start in the notebooks:
```
... starting server cocoMDS4 ...
   ...... (POST http://lab-datalake:8080/open-metadata/admin-services/users/garygeeke/servers/cocoMDS4/instance )
   ...... Response:
          {
              "class": "SuccessMessageResponse",
              "relatedHTTPCode": 200,
              "successMessage": "Wed Jan 15 10:32:26 GMT 2020 cocoMDS4 is running the following services: [Open Metadata Repository Services (OMRS), Connected Asset Services, Asset Catalog OMAS, Asset Consumer OMAS, Community Profile OMAS, Glossary View OMAS, Data Science OMAS, Subject Area OMAS]"
          }
```

It looks as if the configuration is correct, as is the UI -- but Egeria is incorrectly checking whether asset catalog OMAS is running ....

I note that the service is listed as 'Asset Catalog OMAS ' - that additional space looks rogue.

This looks like it's likely a typo. will check the code.

**Comment by planetf1** [comment]
checked - this does affect the 1.3 branch also

**Comment by planetf1** [comment]
The problem is in AssetCatalogServicesInstance.java:38 where we have:

```
        super(description.getAccessServiceName() + " OMAS ", repositoryConnector, auditLog, localServerUserId, repositoryConnector.getMaxPageSize());
```
There is an extra space in this line.

Fixing ..

**Comment by planetf1** [comment]
This fix works in that Asset Catalog OMAS is now available for use.

The UI now correctly can search for file based assets - for example searching on 'csv' after running the asset notebook now works again

However another exception is seen after logging into the UI caused by getting supported types:
```
2020-01-16 09:23:34.313 ERROR 37799 --- [nio-8443-exec-5] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is PropertyServerException{reportedHTTPCode=503, reportingClassName='org.odpi.openmetadata.accessservices.assetcatalog.AssetCatalog', reportingActionDescription='getSupportedTypes', errorMessage='OMAG-COMMON-503-001 A client-side exception was received from API call getSupportedTypes to OMAG Server cocoMDS4 at http://localhost:8081.  The error message was CLIENT-SIDE-REST-API-CONNECTOR-503-002 A client-side exception org.springframework.web.client.HttpServerErrorException$InternalServerError was received by method getSupportedTypes from API call http://localhost:8081/servers/{0}/open-metadata/access-services/asset-catalog/users/{1}/supportedTypes?type={2} to server cocoMDS4 on platform http://localhost:8081.  The error message was 500 : [{"timestamp":"2020-01-16T09:23:34.262+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/servers/cocoMDS4/open-metadata/access-services/asset-catalog/users/faithbroker/supportedTypes"}]', reportedSystemAction='The server has issued a call to the open metadata access service REST API in a remote server and has received an exception from the local client libraries.', reportedUserAction='Look for errors in the local server's console to understand and correct the source of the error.', reportedCaughtException=RESTServerException{reportedHTTPCode=503, reportingClassName='org.odpi.openmetadata.adapters.connectors.restclients.spring.SpringRESTClientConnector', reportingActionDescription='getSupportedTypes', errorMessage='CLIENT-SIDE-REST-API-CONNECTOR-503-002 A client-side exception org.springframework.web.client.HttpServerErrorException$InternalServerError was received by method getSupportedTypes from API call http://localhost:8081/servers/{0}/open-metadata/access-services/asset-catalog/users/{1}/supportedTypes?type={2} to server cocoMDS4 on platform http://localhost:8081.  The error message was 500 : [{"timestamp":"2020-01-16T09:23:34.262+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/servers/cocoMDS4/open-metadata/access-services/asset-catalog/users/faithbroker/supportedTypes"}]', reportedSystemAction='The client has issued a call to the open metadata access service REST API in a remote server and has received an exception from the local client libraries.', reportedUserAction='Review the error message to determine the cause of the error.  Check that the server is running an the URL is correct. Look for errors in the local server's console to understand and correct the cause of the error. Then rerun the request', reportedCaughtException=org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 : [{"timestamp":"2020-01-16T09:23:34.262+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/servers/cocoMDS4/open-metadata/access-services/asset-catalog/users/faithbroker/supportedTypes"}]}, relatedProperties=null}] with root cause
```
I'm inclined to merge the first fix anyway - as the UI now seems to mostly work from the end user's perspective.

But there is a second fix and investigation needed here - perhaps for master only.

**Comment by planetf1** [comment]
Will open a new issue on this other problem
